testsuite/tests/typing-warnings/pr7553.ml

```diff
@@ -0,0 +1,20 @@
+module A = struct type foo end;;
```

Comment:
`;;` needed? And also below.

Reply:
They're here because this file is piped into the REPL not compiled